nilearn/plotting/__init__.py

```diff
@@ -6,6 +6,8 @@
 from ..version import (_import_module_with_version_check,
                        OPTIONAL_MATPLOTLIB_MIN_VERSION)
 
+from .._utils.testing import skip_if_running_nose, is_nose_running
```

Comment:
To avoid polluting the namespace (because that's what the user sees when tab-completing on plotting), I think that we should have the imports inside the relevant function.